src/app/components/content-creator-page/content-creator-page.component.ts

```diff
@@ -166,6 +166,8 @@ export class ContentCreatorPageComponent implements OnInit {
          this.description, this.url,
          this.getLinksFromSubjects(this.selectedSubjects));
 
+      console.log(content);
```

Comment:
remove unnecessary comments and console logs

Reply:
Should have removed all of the console.logs from the ts files in latest commit